.circleci/workflow-queue.sh

```diff
@@ -0,0 +1,181 @@
+#! /bin/bash
```

Comment:
Can this one be deleted now? You can create a new branch if you're worried about losing this work.

Reply:
Deleted.